imap_processing/lo/l1a/lo_l1a.py

```diff
@@ -77,20 +77,22 @@ def lo_l1a(dependency: Path, data_version: str):
 # TODO: This is going to work differently when I sample data.
 #  The data_fields input is temporary.
 def create_datasets(attr_mgr, logical_source, data_fields):
-    """Create a dataset using the populated data classes.
+    """
+    Create a dataset using the populated data classes.
 
     Parameters
     ----------
     attr_mgr : ImapCdfAttributes
-        attribute manager used to get the data product field's attributes
-    logical_source: str
-        The logical source of the data product that's being created
-    data_fields: list[dataclasses.Field]
+        Attribute manager used to get the data product field's attributes.
+    logical_source : str
+        The logical source of the data product that's being created.
+    data_fields : list[dataclasses.Field]
+        List of Fields for data classes.
 
     Returns
     -------
-    xr.dataset
-        dataset with all data product fields in xr.DataArray
+    dataset : xr.dataset
+        Dataset with all data product fields in xr.DataArray.
```

Comment:
```suggestion
    dataset : xarray.Dataset
        Dataset with all data product fields in xarray.DataArray.
```
Same thing. Just saving future trouble